extensions/ql-vscode/test/vscode-tests/cli-integration/data-extension-pack/`

```diff
@@ -0,0 +1,12 @@
+extensions:
```

Comment:
Is the name of this file meant to be a back-tick?

Reply:
Oh...that's odd. This must have been accidentally committed. It can be removed. I'll fix it.